apps/engine.c

```diff
@@ -342,7 +342,7 @@ int engine_main(int argc, char **argv)
             break;
         case OPT_TT:
             test_avail_noise++;
-            /* fall thru */
+            /* FALLTHROUGH */
```

Comment:
I'm a bit surprised, why this is change is necessary,
since there are lots of "fall thru" comments, and only one "FALLTHROUGH".
if this change is necessary then all 136 "fall thru" comments should change.

Reply:
Yeah, my 2nd commit including the modification might be wrong.
The Travis test is still failed for `-Werror=implicit-fallthrough=`.